src/Actions.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import './App.css';
+import Button from "./Button";
+
+let Actions = (props) => {
+
+    return (
+        <div >
+            <div className="row actions">
+                <div className="col text-center">
+                    <Button clickAction={props.startTime} label="START" isDisabled={props.currentAction==="start"}></Button>
```

Comment:
rename to timerState. timerState change to enum. 
button enabled should be decided in app.